cli/functionary/utils.py

```diff
@@ -68,6 +85,8 @@ def format_results(results, title="", excluded_fields=[]):
                 continue
             if first_row:
                 table.add_column(key.capitalize())
+            if "_at" in key:
```

Comment:
Just to be extra safe, this should check that the key specifically **ends with** "_at".  Otherwise you run into something unexpected like:

```python
key = "favorite_attachment"
"_at" in key  # True
```